packages/frontend/src/pages/Playgrounds.svelte

```diff
@@ -22,7 +22,10 @@ function createNewPlayground() {
 }
 </script>
 
-<NavPage title="Playgrounds environments" searchEnabled="{false}">
+<NavPage
+  lastPage="{{ name: 'Playgrounds', path: '/playgrounds' }}"
```

Comment:
This should not be included ? it has been removed in another PR ? Maybe you rebase incorrectly ?